<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLed" name="qLed">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="blink" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>155</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>On/Off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Blink</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>155</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Red</string>
        </property>
        <property name="shortcut">
         <string>Alt+R</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="shortcut">
         <string>Alt+G</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Yellow</string>
        </property>
        <property name="shortcut">
         <string>Alt+Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Grey</string>
        </property>
        <property name="shortcut">
         <string>Alt+E</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Orange</string>
        </property>
        <property name="shortcut">
         <string>Alt+O</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Purple</string>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="shortcut">
         <string>Alt+B</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Shape</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_8">
          <property name="text">
           <string>Circle</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_9">
          <property name="text">
           <string>Square</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_10">
          <property name="text">
           <string>Triangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_11">
          <property name="text">
           <string>Rounded</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>qLed</receiver>
   <slot>setValue(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>qLed</receiver>
   <slot>setBlink(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
